modules/auxiliary/scanner/portmap/portmap_amp.rb

```diff
@@ -20,6 +20,7 @@ def initialize
       'License'     => MSF_LICENSE,
       'References'  =>
         [
+          ['CVE', '2013-5211'], # see also scanner/ntp/ntp_monlist.rb
```

Comment:
The vulnerability and abuse/exploitation is definitely similar, but this is not the same protocol, so IMO referencing this CVE here will cause more problems than it solves.  What am I missing here?